skia/BUILD.gn

```diff
@@ -0,0 +1,832 @@
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+_skia_root = "//third_party/skia"
+
+import("$_skia_root/gn/flutter_defines.gni")
```

Comment:
This should probably refer to a file in y'alls repo

Reply:
Done.